packages/next/src/client/components/react-dev-overlay/_experimental/internal/container/runtime-error/error-provider.tsx

```diff
@@ -93,16 +109,19 @@ export function useErrorHook({
     }
   }, [nextError, isAppDir])
 
-  return {
-    readyErrors,
-    // Total number of errors are based on the priority that
-    // will be displayed. Since build error and root layout
-    // missing tags won't be dismissed until resolved, the
-    // total number of errors may be fixed to their length.
-    totalErrorCount: rootLayoutMissingTags?.length
-      ? 1
-      : !!buildError
-        ? 1
-        : readyErrors.length,
+  return children({ readyErrors, totalErrorCount: readyErrors.length })
+}
+
+const NonRuntimeErrorProvider = ({ children, state }: Props) => {
```

Comment:
it's not really a react provider, we can just name it to BuildError or with some suffix